.prow/scripts/test-end-to-end.sh

```diff
@@ -0,0 +1,212 @@
+  port: 6565
+  enable-reflection: true
+
+feast:
+  version: 0.3
+  jobs:
+    runner: DirectRunner
+    options: {}
+    metrics:
+      enabled: false
+
+  stream:
+    type: kafka
+    options:
+      bootstrapServers: localhost:9092
+      replicationFactor: 1
+      partitions: 1
+
+spring:
+  jpa:
+    properties.hibernate.format_sql: true
+    hibernate.naming.physical-strategy=org.hibernate.boot.model.naming: PhysicalNamingStrategyStandardImpl
+    hibernate.ddl-auto: update
+  datasource:
+    url: jdbc:postgresql://localhost:5432/postgres
+    username: postgres
+    password: password
+
+management:
+  metrics:
+    export:
+      simple:
+        enabled: false
+      statsd:
+        enabled: false
+EOF
+
+nohup java -jar core/target/feast-core-0.3.0-SNAPSHOT.jar \
+  --spring.config.location=file:///tmp/core.application.yml \
+  &> /var/log/feast-core.log &
+sleep 20
+tail -n10 /var/log/feast-core.log
+
+echo "
+============================================================
+Starting Feast Online Serving
+============================================================
+"
+# Start Feast Online Serving in background
+cat <<EOF > /tmp/serving.store.redis.yml
+name: serving
+type: REDIS
+redis_config:
+  host: localhost
+  port: 6379
+subscriptions:
+  - name: .*
+    version: ">0"
+EOF
+
+cat <<EOF > /tmp/serving.online.application.yml
+feast:
+  version: 0.3
+  core-host: localhost
+  core-grpc-port: 6565
+
+  tracing:
+    enabled: false
+
+  store:
+    config-path: /tmp/serving.store.redis.yml
+    redis-pool-max-size: 128
+    redis-pool-max-idle: 16
+
+  jobs:
+    staging-location: gs://feast-templocation-kf-feast/staging-location
+    store-type:
+    store-options: {}
+
+grpc:
+  port: 6566
+  enable-reflection: true
+
+spring:
+  main:
+    web-environment: false
+EOF
+
+nohup java -jar serving/target/feast-serving-0.3.0-SNAPSHOT.jar \
+  --spring.config.location=file:///tmp/serving.online.application.yml \
+  &> /var/log/feast-serving-online.log &
+sleep 15
+tail -n10 /var/log/feast-serving-online.log
+
+echo "
+============================================================
+Installing Python 3.7 with Miniconda and Feast SDK
+============================================================
+"
+# Install Python 3.7 with Miniconda
+wget -q https://repo.continuum.io/miniconda/Miniconda3-4.7.12-Linux-x86_64.sh \
+   -O /tmp/miniconda.sh
+bash /tmp/miniconda.sh -b -p /root/miniconda -f
+/root/miniconda/bin/conda init
+source ~/.bashrc
+
+# Install Feast Python SDK and test requirements
+pip install -q sdk/python
+pip install -qr tests/e2e/requirements.txt
+
+echo "
+============================================================
+Running end-to-end tests with pytest at 'tests/e2e'
+============================================================
+"
+# Default artifact location setting in Prow jobs
+LOGS_ARTIFACT_PATH=/logs/artifacts
+
+ORIGINAL_DIR=$(pwd)
+cd tests/e2e
+
+set +e
+pytest --junitxml=${LOGS_ARTIFACT_PATH}/python-sdk-test-report.xml
```

Comment:
I think we need to make pytest more verbose so that we can see which tests have passed. Perhaps this would help: http://doc.pytest.org/en/latest/usage.html#detailed-summary-report

Reply:
Yeah sounds good